internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanger/qualys/auth/StandardAuthenticationRecordBuilder.java

```diff
@@ -0,0 +1,120 @@
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanmanger.qualys.auth;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.wso2.security.tools.scanmanager.scanners.common.util.XMLUtil;
+import org.wso2.security.tools.scanmanager.scanners.qualys.QualysScannerConstants;
+import org.wso2.security.tools.scanmanager.scanners.qualys.util.RequestBodyBuilder;
+
+import java.io.StringWriter;
+import java.util.Arrays;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+/**
+ * This class is to represent StandardAuthenticationRecordBuilder type of WebAppAuthenticationRecordBuilder
+ * for Qualys scan.
+ */
+public class StandardAuthenticationRecordBuilder implements WebAppAuthenticationRecordBuilder {
+
+    // Username for the StandardAuthenticationRecordBuilder type.
+    private char[] username;
+
+    // Password for the StandardAuthenticationRecordBuilder type.
+    private char[] password;
+
+    public StandardAuthenticationRecordBuilder(char[] userName, char[] password) {
+        this.username = Arrays.copyOf(userName, userName.length);
+        this.password = Arrays.copyOf(password, password.length);
+    }
+
+    @Override public String buildAuthRequestBody(String appID)
+            throws TransformerException, ParserConfigurationException {
+        String standardAuthRecordRequestBody;
+        DocumentBuilderFactory dbf = XMLUtil.getSecuredDocumentBuilderFactory();
+        DocumentBuilder builder = dbf.newDocumentBuilder();
+        Document doc = builder.newDocument();
+
+        Element root = doc.createElement(QualysScannerConstants.SERVICE_REQUEST);
+        doc.appendChild(root);
+
+        Element data = doc.createElement(QualysScannerConstants.DATA);
+        root.appendChild(data);
+
+        Element webAppAuthRecord = doc.createElement(QualysScannerConstants.WEB_APP_AUTH_RECORD);
+        data.appendChild(webAppAuthRecord);
+
+        Element authRecordName = doc.createElement(QualysScannerConstants.NAME_KEYWORD);
+        authRecordName.appendChild(
+                doc.createTextNode("Standard Authentication for " + appID + " : " + RequestBodyBuilder.getDate()));
+        webAppAuthRecord.appendChild(authRecordName);
+
+        Element formRecord = doc.createElement(QualysScannerConstants.FORM_RECORD);
+        webAppAuthRecord.appendChild(formRecord);
+
+        Element type = doc.createElement(QualysScannerConstants.TYPE_KEYWORD);
+        type.appendChild(doc.createTextNode(QualysScannerConstants.STANDARD_AUTH));
+        formRecord.appendChild(type);
+
+        //        Element sslOnly = doc.createElement(QualysScannerConstants.SSL_ONLY);
```

Comment:
Is the necessary? If you want to retain this, shall we add a comment explaining why this is commented. Also better to remove leading spaces.